src/Mutator/Boolean/TrueValue.php

```diff
@@ -60,6 +66,21 @@ protected function mutatesNode(Node $node): bool
             return false;
         }
 
-        return $node->name->toLowerString() === 'true';
+        if ($node->name->toLowerString() !== 'true') {
+            return false;
+        }
+
+        $parentNode = $node->getAttribute(ParentConnectorVisitor::PARENT_KEY);
+        $grandParentNode = $parentNode !== null ? $parentNode->getAttribute(ParentConnectorVisitor::PARENT_KEY) : null;
+
+        if (!$grandParentNode instanceof Node\Expr\FuncCall || !$grandParentNode->name instanceof Node\Name) {
+            return true;
+        }
+
+        $resultSettings = array_merge(self::DEFAULT_SETTINGS, $this->config->getMutatorSettings());
```

Comment:
What about keeping the `$config` private, and instead exposing a protected `getSettings` method in the abstract Mutator class. Then we could do this here.
```diff
- $this->config->getMutatorSettings()
+ $this->getSettings()
```

Reply:
sounds good, will do it